src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  health = 100;
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  static get alive() {
+    return this.alive;
+  }
+  static set alive(aliveAnimals) {
+    this.alive = aliveAnimals;
+  }
```

Comment:
The getter and setter for the static `alive` property are unnecessary and can be removed. The static property `alive` is directly accessible and can be manipulated without these.

Reply:
Remove it please